### PR TITLE
Update behavior tests

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?

As the title says.
- Updating to use `@vtex/test-tools`

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/OrderBy.test.js [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
-    return render(<OrderBy {...props} />)
+    return { ...render(<OrderBy {...props} />), getLinkProps }
   }
 
+  it('should shown dropdown box on mobile mode', () => {
+    const { container } = renderComponent()(true)
+
+    let dropdownContainer = container.querySelector('.dropdownMobile > .dn')
+    const button = container.querySelector('button')
+    // Expect Dropdown container be redered with display none
```
Comment (klzns) [inline react/__tests__/OrderBy.test.js]: typo
```suggestion
    // Expect Dropdown container be rendered with display none
```

Comment (klzns) [reply react/__tests__/OrderBy.test.js]: Might use `.not.toBeVisible()`

https://github.com/gnapse/jest-dom#tobevisible

File: react/__tests__/OrderBy.test.js [hunk 2]
```diff
@@ -1,22 +1,54 @@
...
+    // Expect Dropdown container be redered without display none
+    expect(dropdownContainer).toBeNull()
+  })
+
+  it('should call getLinkProps', () => {
+    const { container, getLinkProps } = renderComponent()(true)
+
+    const button = container.querySelector('button')
+    fireEvent.click(button)
+    // Expect Dropdown container be redered without display none
```
Comment (klzns) [inline react/__tests__/OrderBy.test.js]:
```suggestion
    // Expect Dropdown container be rendered without display none
```

File: react/__tests__/SearchFooter.test.js [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
   }
 
   it('should be rendered', () => {
     const { asFragment } = renderComponent()
     expect(asFragment()).toBeDefined()
   })
 
+  it('should return link', () => {
+    const { container, getLinkProps } = renderComponent()
+    const mainDiv = container.querySelector('.ph2.pointer.c-on-base')
```
Comment (klzns) [inline react/__tests__/SearchFooter.test.js]: Use test-id instead or query for a text.

This is really helpful:
https://testing-library.com/docs/guide-which-query#docsNav

**Comment by theomoura** [comment]
Could you add travis coverage?

File: react/__tests__/OrderBy.test.js (hunk 1 above)
Comment (JuanAlmeida12) [reply react/__tests__/OrderBy.test.js]: Don't work `toBeVisible()` looks the visibility prop and we change `display` value.

**Pull request merged**